app/controllers/JourneyStartController.scala

```diff
@@ -29,12 +30,13 @@ import scala.concurrent.{ExecutionContext, Future}
 
 class JourneyStartController @Inject()(customsSessionCacheConnector: CustomsSessionCacheConnector,
                                        identify: IdentifierAction,
+                                       checkEmailIsVerified: EmailAction,
```

Comment:
Better to rename it like emailVerification or something on those lines as checkEmailIsVerified is not that meaningful?

Reply:
this naming is what has been set for other repos.. some keeping things consistent